packages/server/src/ChunkExtractor.js

```diff
@@ -171,6 +171,11 @@ function isValidChunkAsset(chunkAsset) {
   return chunkAsset.scriptType && !HOT_UPDATE_REGEXP.test(chunkAsset.filename)
 }
 
+const CSS_FILE = /\.css$/
```

Comment:
Rather than testing if all files are CSS, shouldn't it be tested if at least one file is JavaScript?

Reply:
Right, I will modify Regexp and test js.